src/cli/index.js

```diff
@@ -75,6 +75,10 @@ commander.option(
   '--no-emoji',
   'disable emoji in output',
 );
+commander.option(
+  '-s, --silent',
+  'raw script output',
```

Comment:
nit: `silence Yarn console logs` would be closer to correct.
Technically it is not doing anything to raw output.